<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choseAddMotor</class>
 <widget class="QWidget" name="choseAddMotor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>86</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>176</width>
    <height>86</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>176</width>
    <height>86</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chose Motor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MotorTesting.qrc">
    <normaloff>:/icon/icon/AddNew.png</normaloff>:/icon/icon/AddNew.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbMotor">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Motor:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbMotor"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MotorTesting.qrc"/>
 </resources>
 <connections/>
</ui>
